pkg/scheduler/scheduler.go

```diff
@@ -432,17 +435,20 @@ func (s *Scheduler) doScheduleBinding(namespace, name string) error {
 		metrics.BindingSchedule(string(ScaleSchedule), metrics.SinceInSeconds(start), err)
 		return err
 	}
-	// TODO(dddddai): reschedule bindings on cluster change
-	if s.allClustersInReadyState(rb.Spec.Clusters) {
-		klog.Infof("Don't need to schedule ResourceBinding(%s/%s)", namespace, name)
-		return nil
+	exist, ready := s.allClustersInReadyState(rb.Spec.Clusters)
```

Comment:
I think we can add some methods to `Snapshot` might helps, like:
- func (s *Snapshot) ClusterAllExist(clusters []string) bool
- func (s *Snapshot) ClusterAllInReadyState(clusters []string) bool

Reply:
But it has to traverse clusters **twice** if we divide it into two functions, so I didn't do so

Reply:
I know. Fine, it's a tradeoff.